dev/WindowsAppRuntime_BootstrapDLL/MddBootstrapAutoInitializer.cpp

```diff
@@ -29,6 +29,12 @@ namespace Microsoft::Windows::ApplicationModel::DynamicDependency::Bootstrap
             const HRESULT hr{ ::MddBootstrapInitialize(c_majorMinorVersion, c_versionTag, c_minVersion) };
             if (FAILED(hr))
             {
+                if (IsDebuggerPresent()) {
+                    DebugBreak();
+                }
+                else {
+                    MessageBox(NULL,L"You need to run the Windows App Runtime Installer and its MSIX components", L"MddBootstrap", 0x00000010L);
```

Comment:
This is consistent with what I suggested during initial planning with @DrusTheAxe @zaryaf. This would align bootstrap-required app failures with common .NET runtime-required app failures.

I think the message should be reworded to use as much prior art as possible, so power users+ instinctively know what to do next.

Example:
![image](https://user-images.githubusercontent.com/475132/160009131-e737666f-930c-4fee-a190-a828cd0aae0b.png)

Reply:
Beware without a fusion manifest, the message box will be unthemed/ugly. It may or may not get dark mode treatment angering users.

Reply:
Agreed about rewording it, we can say something like: "This application requires installing the Windows App Runtime and its MSIX components."  
At least, can we add a link to the page to install it? 
https://docs.microsoft.com/en-us/windows/apps/windows-app-sdk/downloads

> Beware without a fusion manifest, the message box will be unthemed/ugly. It may or may not get dark mode treatment angering users.

At this point, there is neither WinUI3 nor OS XAML. So we can only delegate on Win32 dialogs, so I hope that at some points all these dialogs support dark more too.

Reply:
@riverar yes, that was my opinion too - model this as closely as possible on the precedent set for the CLR and CRT message boxes

Reply:
Let's replace "you need to" with "please verify" language.  We actually don't have certainty that the WAR isn't installed here, and misdiagnosing can be even more frustrating to users.

Reply:
The title should be "[application_exe] - This application could not be started" versus "MddBootstrap" which doesn't point to the app and has little meaning for end users.